spec/models/job_spec.rb

```diff
@@ -30,9 +30,14 @@
     end
 
     it "allows run for any ref in refs params" do
-      job = FactoryGirl.create :job, project: project, refs: "master, staging"
+      job = FactoryGirl.create :job, project: project, refs: "master, staging, tags/testing*"
```

Comment:
Line is too long. [93/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.